src/Tribe/Views/V2/Views/Day_View.php

```diff
@@ -214,6 +215,47 @@ protected function sort_events( $events ) {
 
 	}
 
+	/**
+	 * Creates a HTML link and "fast forward" message to append to the "no events found" message.
+	 *
+	 * @param bool  $canonical         Whether to return the canonical (pretty) version of the URL or not.
+	 * @param array $passthru_vars     An optional array of query variables that should pass thru the method untouched
+	 *                                 in key and value.
+	 * @return string                  The html link and message.
+	 */
+	public function fast_forward( $canonical = false, array $passthru_vars = [] ) {
+		if ( ! empty( $this->context->get( 'tribe-bar-date' ) ) ) {
```

Comment:
The `event_date` Context location is already looking up `tribe-bar-date` before; see [here](https://github.com/moderntribe/the-events-calendar/blob/620e3f295467ad45a9bb7121ced8531c36ba2005/src/Tribe/Service_Providers/Context.php#L87-L97)
Furthermore, the `tribe-bar-date` Context location does not exist.

I would change this whole block to `$date = $this->context->get( 'event_date', $this->context->get( 'today' ) );`